docs/build/tutorials/smart-contract-development/account-abstraction/custom-aa-tutorial.md

```diff
@@ -944,8 +944,8 @@ export default async function (hre: HardhatRuntimeEnvironment) {
 :::tip
 
 - zkSync has different address derivation rules from Ethereum.
-- Always use the [`createAddress`](../../../sdks/js/utils.md#createaddress) and [`create2Address`](../../../sdks/js/utils.md#create2address) utility functions of the `zksync-ethers` SDK.
-- Read the documentation for more information on [address derivation differences between Ethereum and zkSync](../../../developer-reference/differences-with-ethereum.md).
+- Always use the [`createAddress`](../../../sdks/js/utils.md#createaddress) and [`create2Address`](../../../sdks/js/utils.md#create2address) utility functions of the `zksync-ethers` SDK when looking to determine a contract's address.
+- Read the documentation for more information on [address derivation differences between Ethereum and zkSync](<[../../../developer-reference/differences-with-ethereum.md](https://docs.zksync.io/build/developer-reference/differences-with-ethereum.html#create-create2)>).
```

Comment:
```suggestion
- Read the documentation for more information on [address derivation differences between Ethereum and zkSync](<[../../../developer-reference/differences-with-ethereum.md](https://docs.zksync.io/build/developer-reference/differences-with-ethereum.md#create-create2)>).
```

this breaks the CI job 😉

Reply:
We should be adding links directly to the header within the Markdown file and not use reference links (especially when this is already a link)
```suggestion
- Read the documentation for more information on [address derivation differences between Ethereum and zkSync](../../../developer-reference/differences-with-ethereum.md#create-create2).
```